From af2d2ac7439619629700ee3d985cafeb084b058a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Tue, 14 Jul 2020 18:05:32 +0200
Subject: [PATCH 09/14] Skip RDS socket tests that hang frequently

---
 Lib/test/test_socket.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Lib/test/test_socket.py b/Lib/test/test_socket.py
index e0e0f2437b..557d6945fb 100644
--- a/Lib/test/test_socket.py
+++ b/Lib/test/test_socket.py
@@ -2250,6 +2250,7 @@ def testSocketBufferSize(self):
 
 
 @unittest.skipUnless(HAVE_SOCKET_RDS, 'RDS sockets required for this test.')
+@unittest.skip('Hangs frequently')
 class RDSTest(ThreadedRDSSocketTest):
 
     def __init__(self, methodName='runTest'):
-- 
2.38.1

